tools/msh2vtu.xml

```diff
@@ -0,0 +1,103 @@
+<tool id="msh2vtu" name="GMSH_To_VTU_Converter" version="0.1.0+galaxy0" python_template_version="3.5" profile="21.05">
+    <description>
+        Convert GMSH mesh files to VTU format using various options for mesh processing.
+    </description>
+    <requirements>
+        <requirement type="package" version="3.6">python</requirement>
+        <requirement type="package" version="0.5">meshio</requirement>
+        <requirement type="package" version="0.1">numpy</requirement>
+        <requirement type="package" version="0.1">ogs</requirement>
```

Comment:
Can you use a version macro token for this and the version of the tool itself. You can search for `TOOL_VERSION` is the tools of this repo for examples.